types/mui-datatables/index.d.ts

```diff
@@ -1,4 +1,4 @@
-// Type definitions for mui-datatables 2.14
+// Type definitions for mui-datatables 3.0
```

Comment:
```suggestion
// Type definitions for mui-datatables 3.1
```
Let's use the latest one for review, sounds OK?